src/Servers/Kestrel/tools/CodeGenerator/CodeGenerator.csproj

```diff
@@ -9,6 +9,7 @@
 
   <ItemGroup>
     <Compile Include="$(KestrelSharedSourceRoot)KnownHeaders.cs" />
+    <Compile Include="..\..\Core\src\PseudoHeaderNames.cs" />
```

Comment:
PseudoHeaderNames is also needed by `src\Servers\HttpSys\test\FunctionalTests\Microsoft.AspNetCore.Server.HttpSys.FunctionalTests.csproj`.

Move it to https://github.com/dotnet/aspnetcore/tree/main/src/Shared